### ISSUE TITLE
ODRL IDS-Profile.ttl still valid for the DSP?

### ISSUE THREAD

**Issue by matgnt** [International-Data-Spaces-Association/ids-specification#178]
Is the ODRL profile created for IDS still valid in the DSP?
I'm especially interested in the leftOperand fields

https://docs.internationaldataspaces.org/dataspace-protocol/
"Other Resources"

![Screenshot from 2023-11-22 15-17-05](https://github.com/International-Data-Spaces-Association/ids-specification/assets/198707/a6fe6472-eb5d-480b-913f-54e1ae73549a)

From the gitbook page there is a link into those files but it is not mentioned anywhere else in the spec.

Thanks

--
Matthias Binzer

**Comment by arnoweiss** [comment]
I think this issue can be closed as the profile appears to be managed here [1] and there's no reference in the spec to it.

[1] https://github.com/International-Data-Spaces-Association/ODRL-profile-for-IDS

**Comment by matgnt** [comment]
Hi @arnoweiss 
no, there is no reference in the spec to it, but in the published version as you can see in the screenshot above. Therefore, for the reader of the published version it looks like it belongs together, but the spec itself never touches the topic of the profile - and it has never been discussed in the group - at least not during the year before the DSP has been released as 0.8.

So I would say the question remains valid. Is this profile relevant for DSP or not. If yes, it should be properly referenced somewhere in the spec. If not, the published version should not link to it in its header navigation bar.

**Comment by arnoweiss** [comment]
This is likely just an issue of publication layout and should disappear when IDSA and EDWG documents are more separated. I think it's implausible that could be interpreted as a normative reference.

**Comment by matgnt** [comment]
Yes, it is a publication layout issue and in case the spec is published somewhere else, this is resolved.
